apps/docs/components/Utils/ThemeProvider/ThemeSwitch/ThemeSwitchIcon/ThemeSwitchIcon.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { type Meta, type StoryObj } from "@storybook/react";
+
+import { Icon } from "./ThemeSwitchIcons";
+
+const meta: Meta<typeof Icon> = {
+    title: "Component/Theme Switch/Icon",
+    component: Icon
+};
+
+export default meta;
+type Story = StoryObj<typeof Icon>;
+
+export const Sun: Story = {
```

Comment:
Should we name these Sun and Moon? If we ever change the icon the story name would not make that much sense.